generate.sh

```diff
@@ -2,7 +2,37 @@
 
 srcdir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )
 
+ask1() { local yn; echo -n "$* [Y/n] "; read -r -n 1 yn; [[ "$yn" =~ [Yy] ]] && echo && return 0; [ -z "$yn" ] && return 0; echo; return 1; }
+
 for f in "$srcdir"/{,.}*.mustache; do
-    echo "Generating $(basename "$f" .mustache)..."
-    mustache meta.yml "$f" > "$(basename "$f" .mustache)"
+    target=$(basename "$f" .mustache)
+    case "$target" in
+        coq.opam)
+            mustache='{{ shortname }}'
+            shortname=$(mustache meta.yml - <<<"$mustache")
```

Comment:
Unfortunately, no, that's not portable enough (or we should start saying that a specific implementation of mustache is needed). On the mustache-go implementation, this gives "Error: open -: no such file or directory". I had tried something similar, `mustache meta.yml <(<<< {{shortname}} )`, which did not produce any error but still had the problem that mustache-go doesn't generate the output on stdout but rather create the new filename based on the extension of the initial mustache template. That being said, I realize that this is problematic for the rest of the changes to this script as well. Maybe we should at least exclude some implementations...

Reply:
Hi @Zimmi48, so I reverted the last two commits that were using stdin in place of a regular temp file… can you test again?

Reply:
(obviously I'll do a force-push later on to trim the PR)

Reply:
> I'll do a force-push later on to trim the PR

done now actually…